float32 throttle
float32 steering_angle
float32 brake
bool is_manual